packages/slice-machine/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@slicemachine/manager": "workspace:^",
+    "react-hotkeys-hook": "4.4.1",
```

Comment:
We should not add it to dependencies, I guess. It's something that is internal for us to manage. Then it's like any other dependencies we have in devDependencies. If it's here, it could be mixed with our user's deps.

Maybe @bapmrl you can confirm on that since you did the changes on that part.